static/src/javascripts/projects/common/modules/commercial/user-features.js

```diff
@@ -224,13 +224,25 @@ const getDaysSinceLastOneOffContribution = (): number | null => {
     return dateDiffDays(lastContributionDate, Date.now());
 };
 
-// in last six months
-const isRecentOneOffContributor = (): boolean => {
+// defaults to last six months
+const isRecentOneOffContributor = (holidayDays: number = 180): boolean => {
     const daysSinceLastContribution = getDaysSinceLastOneOffContribution();
     if (daysSinceLastContribution === null) {
         return false;
     }
-    return daysSinceLastContribution <= 180;
+    return daysSinceLastContribution <= holidayDays;
+};
+
+// true if the user is in the first month after ask-free holiday
+const isPostHolidayOneOffContributor = (holidayDays: number = 180): boolean => {
```

Comment:
For now we're sticking with 6 months, but we plan to play with this length in future so I've made it configurable